client/src/annoMatrix/annoMatrix.ts

```diff
@@ -10,45 +10,65 @@ import {
   _getWritableColumns,
 } from "./schema";
 import { indexEntireSchema } from "../util/stateManager/schemaHelpers";
-import { _whereCacheGet, _whereCacheMerge } from "./whereCache";
+import { _whereCacheGet, _whereCacheMerge, WhereCache } from "./whereCache";
 import _shallowClone from "./clone";
-import { _queryValidate, _queryCacheKey } from "./query";
+import { _queryValidate, _queryCacheKey, Query } from "./query";
+import {
+  AnnotationColumn,
+  Category,
+  DataFrame,
+  Field,
+  GCHints,
+  LayoutColumn,
+  Schema,
+} from "../common/types/entities";
 
 const _dataframeCache = dataframeMemo(128);
 
-export default class AnnoMatrix {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-  public isView: any;
+interface Cache {
+  [Field.obs]: Dataframe;
```

Comment:
These are incorrect.  Timmy's `DataFrame` is a sub-object inside the schema (and which should _really_ have a different name!).

The four fields inside the Cache should contain instances of `class DataFrame` as defined in `util/dataframe` (which I am currently working to add types too).